vibin/mediaservers/mediaserver.py

```diff
@@ -209,7 +210,7 @@ def ids_from_filename(
 
     @abstractmethod
     def get_path_contents(
-        self, path: str
+        self, path: Path
```

Comment:
Thanks for cleaning up this `Path` stuff.

Reply:
TBH I was pleasantly surprised to find Python now has type hinting, it didn't last time I used it.